.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -241,24 +157,6 @@ jobs:
         with:
           name: build_files
           path: /home/runner/work/oppia/
-      - name: Unzip build files
-        if: steps.download_artifact_1.outcome != 'failure'
-        run: |
-          ls -la
-          unzip build_files.zip
-          rm build_files.zip
-          echo "Files in ./:"
-          ls -la .
-          echo "Files in oppia_tools:"
-          ls -la oppia_tools
-          echo "Files in oppia:"
-          ls -la oppia
-          echo "Files in build:"
-          ls -la oppia/build
-          echo "Files in third_party:"
-          ls -la oppia/third_party
-        working-directory: /home/runner/work/oppia
-        shell: bash
       - name: (Artifact re-build) Cache third_party/static
```

Comment:
What are we caching?

Reply:
forgot it earlier. removed the caching step